cmd/preview/cmdpreview.go

```diff
@@ -91,6 +98,11 @@ func (r *PreviewRunner) RunE(cmd *cobra.Command, args []string) error {
 		r.Destroyer.DryRunStrategy = drs
 	}
 
+	inventoryPolicy, err := flagutils.ConvertInventoryPolicy(r.inventoryPolicy)
```

Comment:
Should there be new unit tests (or other tests) for the new `InventoryPolicy`?

Reply:
We'll address this in future PRs.